examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,213 @@
+# Individual Conditional Expectation
+# ----------------------------------
+# A new kind of partial dependence plot is available: the Individual
+# Conditional Expectation (ICE) plot. It visualizes the dependence of the
+# prediction on a feature for each sample separately with one line per sample.
+# See the :ref:`User Guide <individual_conditional>`
+
+from sklearn.linear_model import BayesianRidge
+from sklearn.datasets import fetch_california_housing
+from sklearn.inspection import plot_partial_dependence
+
+X, y = fetch_california_housing(return_X_y=True, as_frame=True)
+features = ['MedInc', 'AveOccup', 'HouseAge', 'AveRooms']
+est = BayesianRidge()
+est.fit(X, y)
+display = plot_partial_dependence(
+       est, X, features, kind="individual", subsample=50,
+       n_jobs=3, grid_resolution=20, random_state=0
+)
+display.figure_.suptitle(
+    'Partial dependence of house value on non-location features\n'
+    'for the California housing dataset, with BayesianRidge'
+)
+display.figure_.subplots_adjust(hspace=0.3)
+
+##############################################################################
+# New metrics available
+# ---------------------
+# A number of new metric functions are now available, as for example
+# :func:`metrics.top_k_accuracy_score` and :func:`metrics.det_curve`.
+# For a complete list see the `changelog
+# <../../whats_new/v0.24.html#sklearn-metrics>`_.
+
+##############################################################################
+# DecisionTreeRegressor now supports the new 'poisson' splitting criterion 
+# ------------------------------------------------------------------------
+# The integration of Poisson regression estimation continues from version 0.23
+# :class:`~sklearn.tree.DecisionTreeRegressor` now supports a new `'poisson'`
+# splitting criterion. Setting `criterion="poisson"` might be a good choice
+# if your target is a count or a frequency.
+
+from sklearn.tree import DecisionTreeRegressor
+
+n_samples, n_features = 1000, 20
+rng = np.random.RandomState(0)
+X = rng.randn(n_samples, n_features)
+# positive integer target correlated with X[:, 5] with many zeros:
+y = rng.poisson(lam=np.exp(X[:, 5]) / 2)
+X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=rng)
+regressor = DecisionTreeRegressor(criterion='poisson', random_state=0)
+regressor.fit(X_train, y_train)
+
+##############################################################################
+# New documentation improvements
+# ------------------------------
+#
+# New examples and documentation pages
+# have been addeed in a continuous effort
+# to improve the understanding of data science practices.
+# Among others a new section about :ref:`common pitfalls and recommended
+# practices <common_pitfalls>` is now included, and an `example
+# <../model_selection/plot_grid_search_stats.html>`_ illustrating
+# how statistically compare the performance of models
+# evaluated using :class:`~sklearn.model_selection.GridSearchCV`.
```

Comment:
maybe put back the PLS vs PCR example, and add the linear model coefs one? Also a bullet point list might help readability

Reply:
The linear model one has been released in 0.23.

Reply:
I was thinking we could just cheat a little, since it's a useful example. We can precise

> (actually introduced in 0.23)

if we really want to stick to the book, but I don't think anyone would mind